graphene_boilerplate/config.py

```diff
@@ -8,5 +8,5 @@
 PROJECT_NAME = LOGGER_NAME = 'graphene_boilerplate'
 SECRET_KEY = getenv('SECRET_KEY', default='testsecretkey')
 
-SQLALCHEMY_DATABASE_URI = getenv('SQLALCHEMY_DATABASE_URI', default='postgresql://localhost:5432/fullgraph_alchemist')
+SQLALCHEMY_DATABASE_URI = getenv('SQLALCHEMY_DATABASE_URI', default='postgresql://kiska:qwe123qwe@localhost:5432/fullgraph_alchemist')
```

Comment:
I want you to understand that this is in no way production code. We would use env variables if we were a bit more serious with this project